skxray/core/fitting/lineshapes.py

```diff
@@ -385,3 +391,100 @@ def compton(x, compton_amplitude, coherent_sct_energy,
     counts += value
 
     return counts
+
+
+def gamma_dist(bin_values, K, M):
```

Comment:
It is not important to keep K and M capitalized, sure we can change to  κ and θ when we drop 2.7 .